EmpleoDotNet/Views/Shared/_JobListPartial.cshtml

```diff
@@ -9,7 +9,7 @@
             <a href="@Url.Action("Detail", "JobOpportunity" , new { id=UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)})" class="featured">
                 <div class="row">
                     <div class="col-md-1 hidden-sm hidden-xs text-center">
-                        @if (!string.IsNullOrEmpty(jobOpportunity.CompanyLogoUrl))
+                        @if (UrlHelperExtensions.IsImageAvailable(jobOpportunity.CompanyLogoUrl))
```

Comment:
I was thinking about that, I did it that why to  fix the problem with the images that already are in the database. But we can put that validation just when we create a new job Opportunity.
What do you think @anyeloamt ?